src/kbmod/search/LayeredImage.cpp

```diff
@@ -22,6 +22,28 @@ LayeredImage::LayeredImage(std::string path, const PointSpreadFunc& psf) : psf(p
     loadLayers(path);
 }
 
+LayeredImage::LayeredImage(const RawImage& sci, const RawImage& var, const RawImage& msk,
```

Comment:
I'm actually not sure what makes the most sense here. I don't think it would hurt to add them. They don't add overhead, right?

Reply:
No overhead. It is a compiler directive that prohibits implicit casts to match the argument list. It is not always appropriate but (like const) can be helpful to avoid side effects.